test/unit/webcomponent-proto-funcs.js

```diff
@@ -0,0 +1,22 @@
+import reactify from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+let x = 0;
+function createComponent() {
+  const proto = Object.create(HTMLElement.prototype);
+  proto.foo = function() {
+    return 'bar';
```

Comment:
I'd like a test where one of functions returns some internal state from the web component (i.e. to test "this" binding is correct).